third_party/binutils/test_decenc/test_decenc_x86_64.asm

```diff
@@ -106651,6 +106651,27 @@ GLOBAL_LABEL(FUNCNAME:)
         RAW(62) RAW(b2) RAW(5d) RAW(08) RAW(53) RAW(d6)
         RAW(62) RAW(d2) RAW(5d) RAW(08) RAW(50) RAW(d4)
 
+        /* x86_64_xsaves.s */
+        RAW(0f) RAW(c7) RAW(28)
```

Comment:
See above: this file has an existing disabled xsaves test; maybe also xrstors, didn't search that one.

Reply:
Re xrstors: Didn't find that string or similar bytes that are disabled.

Re the disabled xsaves test: that one is subsumed by the new ones I added. I deleted that.